### PR TITLE
yambar: new package

### PR DESCRIPTION
**Pull request by Ogromny**
## Description of package
https://codeberg.org/dnkl/yambar
Modular status panel for X11 and Wayland, inspired by https://github.com/jaagr/polybar

## Installed manifest of package
```
/var/db/kiss/installed/yambar/version
/var/db/kiss/installed/yambar/sources
/var/db/kiss/installed/yambar/manifest
/var/db/kiss/installed/yambar/depends
/var/db/kiss/installed/yambar/checksums
/var/db/kiss/installed/yambar/build
/var/db/kiss/installed/yambar/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man5/yambar.5
/usr/share/man/man5/yambar-tags.5
/usr/share/man/man5/yambar-particles.5
/usr/share/man/man5/yambar-modules.5
/usr/share/man/man5/yambar-modules-xwindow.5
/usr/share/man/man5/yambar-modules-xkb.5
/usr/share/man/man5/yambar-modules-sway.5
/usr/share/man/man5/yambar-modules-sway-xkb.5
/usr/share/man/man5/yambar-modules-script.5
/usr/share/man/man5/yambar-modules-river.5
/usr/share/man/man5/yambar-modules-removables.5
/usr/share/man/man5/yambar-modules-network.5
/usr/share/man/man5/yambar-modules-mpd.5
/usr/share/man/man5/yambar-modules-mem.5
/usr/share/man/man5/yambar-modules-label.5
/usr/share/man/man5/yambar-modules-i3.5
/usr/share/man/man5/yambar-modules-foreign-toplevel.5
/usr/share/man/man5/yambar-modules-cpu.5
/usr/share/man/man5/yambar-modules-clock.5
/usr/share/man/man5/yambar-modules-battery.5
/usr/share/man/man5/yambar-modules-backlight.5
/usr/share/man/man5/yambar-modules-alsa.5
/usr/share/man/man5/yambar-decorations.5
/usr/share/man/man5/
/usr/share/man/man1/yambar.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/doc/yambar/README.md
/usr/share/doc/yambar/LICENSE
/usr/share/doc/yambar/
/usr/share/doc/
/usr/share/applications/yambar.desktop
/usr/share/applications/
/usr/share/
/usr/include/yambar/yml.h
/usr/include/yambar/tag.h
/usr/include/yambar/stride.h
/usr/include/yambar/particle.h
/usr/include/yambar/module.h
/usr/include/yambar/log.h
/usr/include/yambar/decoration.h
/usr/include/yambar/config.h
/usr/include/yambar/config-verify.h
/usr/include/yambar/color.h
/usr/include/yambar/bar/bar.h
/usr/include/yambar/bar/
/usr/include/yambar/
/usr/include/
/usr/bin/yambar
/usr/bin/
/usr/
```

## New package

- [X] Latest upstream version.
- [X] I agree to maintain this package.
- [X] I agree to notify the @kiss-community maintainers if I can no longer maintain this package.

### CODE REVIEW

File: community/yambar/build [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+meson \
+	--prefix=/usr \
+	-Dbackend-x11=disabled \
+	-Dbackend-wayland=enabled \
+	-Dplugin-mpd=disabled \
+	build
+DESTDIR="$1" ninja -C build install
```
Comment (aabacchus) [inline community/yambar/build:9]: Please follow the style guide: https://kisscommunity.bvnf.space/kiss/style-guide/#0600

File: community/yambar/build [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# remove zsh shit
+rm -rf "$1/usr/share/zsh"
+
+# remove man doc if mandoc not installed
+if ! kiss search mandoc 2>/dev/null 1>/dev/null; then
+	rm -rf "$1/usr/share/man"
+fi
```
Comment (aabacchus) [inline community/yambar/build:17]: This kind of pruning should probably be left to a user's $KISS_HOOK; you can still have use for man pages if the mandoc package isn't installed.

**Review by aabacchus — CHANGES_REQUESTED** [review]

File: community/yambar/build (hunk 1 above)
Comment (Ogromny) [reply community/yambar/build]: Thanks, I hadn't see this page

Comment (aabacchus) [reply community/yambar/build]: No worries. The style guide is also in this repo (style-guide.txt)

**Pull request merged**